### ISSUE TITLE
Manage form submissions permission doesn't allow changing status from CP edit submission page

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#763]
**Description**
When a user has **Manage "{form}" submissions** set, they are unable to change the submission status from the CP edit submissions page. Trying to do so results in the following error:
> **Unauthorized**
> 
> User is not permitted to perform this action

**Steps to reproduce**
1. Set a user account with only the **View submissions** setting and a **Manage "{form}" submissions** setting.
2. With that user account, go to a CP edit submission page for a submission from that form.
3. Select a new submission status and save. The above error message displays.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.5.9
- Craft version: Pro 3.7.33
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.9",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.10](https://github.com/verbb/formie/releases/tag/1.5.10)
